common-libs/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/common",
-  "version": "2.3.0",
+  "version": "3.0.0",
```

Comment:
despite this library uses libraries that have to be update with major version, this library (common) doesn't use bloom-vault under the hood, so my opinion **to update it with a minor update**

Reply:
changed